debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/GetNestCommand.java

```diff
@@ -0,0 +1,181 @@
+import com.ibm.j9ddr.tools.ddrinteractive.CommandUtils;
+import com.ibm.j9ddr.tools.ddrinteractive.Context;
+import com.ibm.j9ddr.tools.ddrinteractive.DDRInteractiveCommandException;
+import com.ibm.j9ddr.util.PatternString;
+import com.ibm.j9ddr.vm29.j9.DataType;
+import com.ibm.j9ddr.vm29.j9.ObjectModel;
+import com.ibm.j9ddr.vm29.j9.Pool;
+import com.ibm.j9ddr.vm29.j9.SlotIterator;
+import com.ibm.j9ddr.vm29.j9.walkers.ClassIterator;
+import com.ibm.j9ddr.vm29.pointer.AbstractPointer;
+import com.ibm.j9ddr.vm29.pointer.SelfRelativePointer;
+import com.ibm.j9ddr.vm29.pointer.VoidPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9BuildFlags;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ClassLoaderPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ClassPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9JavaVMPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ModulePointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ObjectPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9PoolPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMClassPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9UTF8Pointer;
+import com.ibm.j9ddr.vm29.pointer.helper.J9ClassHelper;
+import com.ibm.j9ddr.vm29.pointer.helper.J9ClassLoaderHelper;
+import com.ibm.j9ddr.vm29.pointer.helper.J9ObjectHelper;
+import com.ibm.j9ddr.vm29.pointer.helper.J9RASHelper;
+import com.ibm.j9ddr.vm29.pointer.helper.J9UTF8Helper;
+import com.ibm.j9ddr.vm29.tools.ddrinteractive.JavaVersionHelper;
+import com.ibm.j9ddr.vm29.types.U16;
+
+/**
+ * GetNest command Outputs the nest host and all the nest members of the class
+ * 
+ * Example: 
+ * !getnest 0x0000000001919400
+ *   
+ * Example output: 
+ * NestHost: 
+ * ClassIsOwnNestHost !j9class 0x0000000001919400* 
+ * NestMembers:
+ * 
+ */
+public class GetNestCommand extends Command 
+{
+	public GetNestCommand() 
+	{
+		addCommand("getnest", "<j9ClassAddress>", "Output the nest host and all the nest members of the class");
+	}
+
+	public void run(String command, String[] args, Context context, PrintStream out) throws DDRInteractiveCommandException
+	{
+		if (args.length == 0) {
+			printUsage(out);
+			return;
+		}
+		try {
+			J9JavaVMPointer vm = J9RASHelper.getVM(DataType.getJ9RASPointer());
+			if (JavaVersionHelper.ensureJava11AndUp(vm, out)) {
+				String hexAddress = args[0];
+				long address = CommandUtils.parsePointer(hexAddress, J9BuildFlags.env_data64);
+				J9ClassPointer ramClass = J9ClassPointer.cast(address);
+				J9ClassLoaderPointer classLoader = ramClass.classLoader();
+				J9ROMClassPointer romClass = ramClass.romClass();
+				String name = J9UTF8Helper.stringValue(romClass.className());
+				long nestMemberCount = romClass.nestMemberCount().longValue();
+
+				if (romClass.nestHost() == VoidPointer.NULL) {
+					/* If the class has NULL nest host, it is its own nest host */
+					out.printf("NestHost:\n" + "*%s !j9class %s*\nNestMembers:\n", name, hexAddress);
```

Comment:
Can the headings be changed to 'Nest host:' and 'Nest members:', respectively (remember your human audience). Also applies to places below where the same headings are printed.